src/content/en/2021/caching.md

```diff
@@ -1,20 +1,488 @@
+
+Mobile resources with a cache TTL that was considered too short compared to its content age have seen an improvement since 2020. This data is exciting because it hints at the community's growing understanding of appropriately relative caching.
+
+While a cache TTL that is too long may serve stale content, there is no benefit for the end user if it is too short. The connection between cache TTL and content age is slowly closing this gap, moving from 60.2% in 2020 to 54% in 2021. The more attentive we can be towards to content age (i.e: how often we revamp a page's HTML, CSS etc), the more accurately we can set cache limits.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Developers are getting better at setting the cache duration more accurately to the content age, resulting in more responsible, and therefore more effective, caching.
+
+<figure>
+  <table>
+    <tr>
+     <th>Client</th>
+     <th>1st party</th>
+     <th>3rd party</th>
+     <th>Overall</th>
+    </tr>
+    <tr>
+     <td>Desktop</td>
+     <td class="numeric">59.5%</td>
+     <td class="numeric">46.2%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+    <tr>
+     <td>Mobile</td>
+     <td class="numeric">60.1%</td>
+     <td class="numeric">44.7%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+  </table>
+  <figcaption>{{ figure_link(
+    caption="TODO",
+    sheets_gid="1069912023",
+    sql_file="content_age_older_than_ttl_by_party.sql"
+  ) }}</figcaption>
+</figure>
+
+When we split the data between 1st and 3rd party providers, the largest improvements come from 3rd parties where we have a 13.2% improvement. It is highly encouraging to see companies around the world building products for developers that are optimizing caching. It's possible that the developer community's increased attention towards improving performance has encouraged and even incentivized 3rd parties to optimize their caching strategies.
+
+However, the challenge remains for how 1st parties can effectively improve over the coming years.
+
+## Identifying caching opportunities
+
+Based on the **Lighthouse caching TTL** score, we have seen an improvement in pages ranked with a perfect score of 100 increase from 3.3% in 2020 to 4.4% in 2021.
+
+The score reflects whether the pages can benefit from additional caching policy improvements. Even though we are excited to see 31% of pages scoring above the 50 percentile score, a large potential for improvement exists for the 52% of pages that are ranking below the 25 percentile.
+
+{{ figure_markup(
+  image="lighthouse-caching-ttl-scores.png",
+  caption="Distribution of Lighthouse caching TTL scores.",
+  description="TODO",
```

Comment:
description="A bar chart showing the distribution of Lighthouse audit scores for the `uses-long-cache-ttl` for mobile web pages. 36.2% of the responses have a score less than 0.10, 25.6% have a score between 0.10-0.39, 20.3% have a score between 0.40-0.79, and 12.1% have a score between 0.80-0.99. 4.4% have a score of 1 and 1.3% have no score.",